.eslintrc.json

```diff
@@ -1,17 +1,21 @@
 {
   "extends": [
-    "okonet"
+    "okonet",
+    "plugin:node/recommended"
   ],
+  "plugins": ["node"],
   "env": {
     "browser": false,
     "node": true,
     "jest": true,
     "es6": true
   },
   "rules": {
-    "strict": 0,
-    "import/no-extraneous-dependencies": [2, {
-      "devDependencies": ["**/*.spec.js"]
-    }]
+    "strict": "off",
```

Comment:
We should probably enable it and add strict mode everywhere.

Reply:
+1 for strict mode.

Reply:
Seems I can't update it, conflicts with `parserOptions`